### PR TITLE
fix: workspace URL parameters after the user accepts the OAuth authorization

### PR DESCRIPTION
**Pull request by olexii4**
### What does this PR do?
Fix workspace URL parameters after the user accepts the OAuth authorization.

### What issues does this PR fix or reference?
fixes https://issues.redhat.com/browse/CRW-4955

### CODE REVIEW

**Review by tolusha — APPROVED** [review]
Tested, it works as expected.

**Review by akurinnoy — APPROVED** [review]

**Review by ibuziuk — APPROVED** [review]
@dmytro-ndp we also need your approval before merging

**Comment by dmytro-ndp** [comment]
PR has been verified successfully following "_Steps to Reproduce_" from https://issues.redhat.com/browse/CRW-4955
Test screenshot: https://youtu.be/i0xoktfXQTc

Good job, @olexii4 !

**Pull request merged**
